documentation-website/Writerside/snippets/get-started-with-exposed/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Ignore Gradle project-specific cache directory
+.gradle
```

Comment:
It looks like we have no similar `.gitignore` files in many other modules. I expect that `.gradle` and `build` directories should be excluded by root `.gitignore` file (better to check).

Reply:
Seems so! I removed it now.